internal/gengapic/genrest.go

```diff
@@ -144,23 +143,152 @@ func (g *generator) genRESTMethod(servName string, serv *descriptor.ServiceDescr
 	if pf, err := g.pagingField(m); err != nil {
 		return err
 	} else if pf != nil {
-		// TODO(dovs)
-		return nil
+		iter, err := g.iterTypeOf(pf)
+		if err != nil {
+			return err
+		}
+
+		return g.pagingRESTCall(servName, m, pf, iter)
 	}
 
 	switch {
 	case m.GetClientStreaming():
-		// TODO(dovs)
-		return nil
+		return g.noRequestStreamRESTCall(servName, serv, m)
 	case m.GetServerStreaming():
-		// TODO(dovs)
-		return nil
+		return g.serverStreamRESTCall(servName, serv, m)
 	default:
-		// TODO(dovs)
 		return g.unaryRESTCall(servName, m)
 	}
 }
 
+func (g *generator) serverStreamRESTCall(servName string, s *descriptor.ServiceDescriptorProto, m *descriptor.MethodDescriptorProto) error {
+	// Streaming calls will most likely NEVER be supported for REST clients,
+	// but the interface signature must be preserved.
+	// Making sure not to call streaming methods on a REST client, or checking for
```

Comment:
BiDi streaming will definitely never be supported over REST.
Unidirectional streaming may be. 
I would change the wording to "streaming calls are not currently supported..."